### PR TITLE
nexus 3.28.1

### PR DESCRIPTION
**Pull request by SMillerDev**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
  Errno::ENOENT: No such file or directory @ rb_sysopen - nexus-3.28.1/conf/nexus.properties
```

File: Formula/nexus.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 class Nexus < Formula
   desc "Repository manager for binary software components"
   homepage "https://www.sonatype.org/"
-  url "https://sonatype-download.global.ssl.fastly.net/repository/downloads-prod-group/oss/nexus-2.14.19-01-bundle.tar.gz"
-  version "2.14.19-01"
-  sha256 "5bf321e40b219756441fbdb9fa0d270493a25640af67076111ccd151ab007b7e"
+  url "https://github.com/sonatype/nexus-public/archive/release-3.28.1-01.tar.gz"
```
Comment (chenrui333) [inline Formula/nexus.rb:4]: maybe this would be right artifact location?
`http://download.sonatype.com/nexus/3/latest-mac.tgz`

Comment (SMillerDev) [reply Formula/nexus.rb:4]: That's unversioned, we can't use that.

**Comment by SMillerDev** [comment]
It doesn't seem to be picking up the VM options. Maybe I should just always write the file.

**Review by chenrui333 — COMMENTED** [review]
💯

**Pull request closed without merging**